### DISCUSSION TITLE
I want to embed OpneGLView in WinUI, WPF Grid control, etc.

### DISCUSSION THREAD

**Discussion by MAUIHighSchool** [dotnet/Silk.NET#1330] [Q&A] [answered]
I would like to embed OpneGLView in WinUI, WPF's Grid control, etc. and use it like UserControl, is there such a sample?

**Comment by Beyley** [comment, marked as answer]
i do not know of anyone using Silk.NET in WPF, but you can probably do it easily if you can get an OpenGL context and a getprocaddress function

**Reply by MAUIHighSchool** [reply]
You don't have a sample...
I don't know how to implement it.

**Reply by Beyley** [reply]
> You don't have a sample...
> I don't know how to implement it.

it's not something any of the maintainers are focused on, so you are mostly on your own, apologies, if you can find an OpenTK example for it you should be able to modify it to use silk easily

**Reply by MAUIHighSchool** [reply]
I found a sample of WPF, WinUI implementation using Silk.NET.
https://github.com/qian-o/SilkDemo
However, the WGL binding part is not yet implemented in Silk.NET, so I will use OpenTK WGL binding only for that part.
